flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/agg/AggsHandlerCodeGenerator.scala

```diff
@@ -333,6 +333,10 @@ class AggsHandlerCodeGenerator(
 
     val functionName = newName(name)
 
+    // make sure we can get the RuntimeContext properly
+    val openCode = ctx.reuseOpenCode().replaceAll("\\(getRuntimeContext\\(\\)\\)",
```

Comment:
What not introduce a method `getRuntimeContext` in this agg handle function?

Reply:
I thought it's simpler this way, since we only need get the RuntimeContext in the open method. I'll add a `getRuntimeContext` method if you think that's better.

Reply:
I think we should avoid some codes like replace string in code generation.
It may brings some potential bugs in future.